dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -1937,7 +2318,8 @@
                                         <Setter Target="ExtensionGlyph.Opacity" Value="1" />
                                     </VisualState.Setters>
                                 </VisualState>
-                                <VisualState x:Name="HideExtensionGlyph" />
+                                <!--Without commenting this one, glyph is getting invisible when pressing the button-->
+                                <!--<VisualState x:Name="HideExtensionGlyph" />-->
```

Comment:
You also could add another way here. Make the HideExtensionGlyph as same as ShowExtensionGlyph

Reply:
Currently, we don't need ExtensionGlyph any more. Why ExtensionGlyph will influence SelectionAccent?